src/jit/CMakeLists.txt

```diff
@@ -8,6 +8,10 @@ if (CLR_CMAKE_TARGET_ARCH_AMD64 OR (CLR_CMAKE_TARGET_ARCH_I386 AND NOT CLR_CMAKE
   add_definitions(-DFEATURE_SIMD)
 endif ()
 
+if (CLR_CMAKE_TARGET_ARCH_ARM64)
+  add_definitions(-DFEATURE_SIMD)
```

Comment:
why not add it as another "OR" above?

Reply:
You'll also need to alter protononjit\CMakeLists.txt: that file removes FEATURE_SIMD (since x86/x64 define it, but the x86_arm and x64_arm64 altjits currently do not support SIMD). You'll need to move the "remove" into the "I386" section, since the x64_arm64 altjit will support SIMD.

We might need to think about what needs to happen, if anything, to make the altjit work equivalently to the native JIT.

(btw, I presume one of your changes fixes `CILJit::getMaxIntrinsicSIMDVectorLength`)

Reply:
> why not add it as another "OR" above?

If I had trouble with windows compilation, I was thinking about disabling, so the logic above would get complex.

I will move it.

> You'll also need to alter protononjit\CMakeLists.txt

OK

> (btw, I presume one of your changes fixes CILJit::getMaxIntrinsicSIMDVectorLength)
I have one `getMaxIntrinsicSIMDVectorLength` change.  `CILJIT` namespace does not look familiar.  I'll take a look when I get back to the office.